src/main/java/com/school/project/controller/SubjectController.java

```diff
@@ -0,0 +1,40 @@
+package com.school.project.controller;
+
+import com.school.project.model.entities.Subject;
+import com.school.project.service.SubjectService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping
+public class SubjectController {
+
+    @Autowired
+    SubjectService subjectService;
+
+    @PostMapping("/subject")
+    public void createSubject(@RequestBody Subject subject) {
```

Comment:
you won't get Subject entity here, it should be SubjectDto, please check https://github.com/MichelRogov/Accounting/pull/54 or https://github.com/MichelRogov/Accounting/pull/53